package.json

```
@@ -18,24 +18,29 @@
     "LICENSE"
   ],
   "scripts": {
-    "lint": "eslint . && jscs index.js lib/ test/",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "mocha --async-only",
-    "cover": "istanbul cover _mocha --report lcovonly",
-    "coveralls": "npm run cover && istanbul-coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "once": "^1.3.2"
+    "once": "^1.4.0"
   },
   "devDependencies": {
-    "eslint": "^1.7.3",
-    "eslint-config-gulp": "^2.0.0",
-    "expect": "^1.19.0",
-    "istanbul": "^0.4.3",
-    "istanbul-coveralls": "^1.0.3",
-    "jscs": "^2.3.5",
-    "jscs-preset-gulp": "^1.0.0",
-    "mocha": "^2.4.5"
+    "eslint": "^7.32.0",
+    "eslint-config-gulp": "^5.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "expect": "^27.4.2",
+    "mocha": "^8.4.0",
+    "nyc": "^15.1.0"
+  },
+  "nyc": {
+    "reporter": [
+      "lcov",
+      "text-summary"
+    ]
+  },
+  "prettier": {
+    "singleQuote": true
   },
   "keywords": [
     "async",
```

Can you also complete #13 by adding:
```suggestion
    "async",
    "parallel",
    "series",
    "each",
```